tests/shell/shell_recorder/README.md

```diff
@@ -24,7 +24,9 @@ If File is preset but empty a fresh database temp file will be provided for ever
 
 Posix-extended regex is used to check and validate return values and outputs.
 
-Options: 
+**Remark:** Shell Recorder uses the `⏎` symbol as line terminator. This means that you need to use the character `⏎`  (instead of `\n`) if you want to match a line ending in a multiline output.
```

Comment:
Can you describe at which positions this line terminator is accepted, how to enter the character, and make an example how to use it?